src/release/main.d

```diff
@@ -893,7 +931,10 @@ Version autodetectVersions ( Version[] tags )
 
     enforce(detected);
 
-    writefln("We are on branch %s", current);
+    if (rel_ver.type == rel_ver.type.Patch)
+        enforce(!options.pre_release,
+            "Prerelease is only for non-patch releases allowed!");
```

Comment:
I would use "Pre-release" as this is how is mentioned in semver (and is easier to read).